apps/blade/src/app/member/dashboard/page.tsx

```diff
@@ -0,0 +1,17 @@
+import Image from "next/image";
+
+import DashboardComponent from "../components/dashboard-component";
+
+export default function Dashboard() {
+  return (
+    <div>
+      <DashboardComponent />
+      <Image
+        src={"../../../../../blade/public/tk-dashboard-img.svg"}
+        alt="tk-pic"
```

Comment:
Alt messages should probably be a little more descriptive. Something like "A picture of Tech Knight". Avoid acronyms for better accessibility.